Cargo.toml

```diff
@@ -11,11 +11,16 @@ members = [
     "rust-connectors/sinks/dynamodb/",
     "rust-connectors/sinks/kafka",
     "rust-connectors/models/fluvio-model-postgres",
+    "rust-connectors/models/fluvio-model-sql",
     "rust-connectors/utils/mocks/http-json-mock/",
     "rust-connectors/utils/connector-run",
     "rust-connectors/utils/test-connector",
     "rust-connectors/utils/fluvio-smartstream-map/",
+    "smartmodules/json-sql",
     "examples/github-stars/github-stars-smartmodule-map/",
     "examples/github-stars/slack-display-smartmodule-map/",
 ]
 resolver = "2"
+
+[patch.crates-io]
+fluvio-smartmodule = { git = "https://github.com/infinyon/fluvio" }
```

Comment:
It'd probably be good to have the `rev` pointing to the exact commit so it won't break as easily. This has been sitting in review long enough that I don't feel strongly about it.

Reply:
Very good point. Thanks. Done.